sky/utils/cli_utils/status_utils.py

```diff
@@ -243,6 +246,31 @@ def _get_price(cluster_status):
     return price_str
 
 
+def _get_total_cost(cluster_status):
+    handle = cluster_status['handle']
+    metadata = cluster_status['metadata']
+    cluster_name = cluster_status['name']
+    status = cluster_status['status']
+
+    if handle and metadata:
+        usage_intervals = metadata['usage_intervals']
+        cost_before_start = global_user_state.get_cost_for_usage_intervals(
+            handle, usage_intervals)
+
+        start_time = global_user_state.get_cluster_launch_time(cluster_name)
+        query_time = int(time.time())
+        cost_to_query = 0
+
+        if status == global_user_state.ClusterStatus.UP:
+            cost_to_query = global_user_state.get_cost_for_usage_intervals(
+                handle, [(start_time, query_time)])
+
+        total_cost = cost_before_start + cost_to_query
```

Comment:
Why don't we have these in the `get_cost_for_cluster`? Also, if we save the `start_time` in the `metadata['usage_intervals']`, it can becomes:
```
[(start1, end1), (start2, end2), (start3, None)] 
```
In that case, we don't need to have many conditions here, but just use:
```
query_time = ...
for i, (start_time, end_time) in enumerate(usage_intervals):
    start_time, end_time = start_time, end_time
    if end_time is None:
         assert i == len(usage_intervals) - 1, i
         end_time = query_time
    total_duration += end_time - start_time
```